yarn-project/types/src/event_logs.ts

```diff
@@ -2,19 +2,19 @@ import { BufferReader, serializeBufferToVector } from '@aztec/foundation/seriali
 import { randomBytes } from 'crypto';
 
 /**
- * Data container of unverified data corresponding to one L2 block.
+ * Data container of encrypted logs corresponding to one L2 block.
```

Comment:
This class will be used for unencrypted logs as well in the future so I would not mention the word "encrypted" anywhere here.